draft/spec/inventory_schema.json

```diff
@@ -80,7 +80,7 @@
     "required": [ "digestAlgorithm", "head", "id", "manifest", "type", "versions" ],
     "definitions": {
         "digests_to_files": {
-            "description": "Object pattern used for `manifest`, `state` and within `fixity` objects. Keys are digests which MUST be hex strings with chars 0-9, a-f or A-F, values are arrays of file paths (non-empty strings). This pattern permits digests that are 32, 40, 64, or 128 hex digits long (matching md5, sha1, sha256, and sha512 respectively) but does not check that the length is correct for the type",
+            "description": "Object pattern used for `manifest`, `state` and within `fixity` objects. Keys are digests which MUST be hex strings with chars 0-9, a-f or A-F, values are arrays of file paths (non-empty strings). This pattern permits digests that are 32, 40, 64, or 128 hex digits long (matching md5, sha1, sha256, and sha512 / blake2b-512 respectively) but does not check that the length is correct for the type",
```

Comment:
Is there a reason for using `/ blake2b-512` instead of `and blake2b-512 `?

Reply:
Because the 'respectively' list aligns with 32, 40, 64, and 128, and both SHA512 and Blake2B use 128 hex digits. 

"md5, sha1, sha256, and sha512 or blake2b-512" is correct, but weird.

"md5, sha1, sha256, and sha512, and blake2b-512" is incorrect since it presents five options against a four option list.

Since this is just a description in the JSON Schema file, I didn't think too long about it, though.